frontend/src/metabase/containers/SaveQuestionModal.tsx

```diff
@@ -182,6 +201,7 @@ export default class SaveQuestionModal extends Component {
   }
 }
 
+// @ts-expect-error provide correct Props type
 const SaveTypeInput = ({ field, originalQuestion }) => (
```

Comment:
Could we specify props?

Reply:
If you mean a type is coming from `metabase-types/types`, it should be fine to use it when typing old components. Ideally, we'd want to use `metabase-types/api`  for new code and refactor the old code to use them as well